iceoryx_examples/request_response_in_c/server_c_basic.c

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceoryx_binding_c/request_header.h"
+#include "iceoryx_binding_c/response_header.h"
+#include "iceoryx_binding_c/runtime.h"
+#include "iceoryx_binding_c/server.h"
+#include "request_and_response_c_types.h"
+#include "sleep_for.h"
+
+#include <signal.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+
+bool keepRunning = true;
+const char APP_NAME[] = "iox-c-request-response-server-basic";
+
+void sigHandler(int signalValue)
+{
+    (void)signalValue;
```

Comment:
Can be avoided by omitting the parameter name.

Reply:
sorry this is a c20 extensions - no can do

Reply:
Oh well, fine then. Is it really 20? ... at least I am already used to it.